api/middleware/auth.js

```diff
@@ -2,7 +2,7 @@ const logger = require('../logger')('auth middleware');
 
 const canCache = {};
 
-module.exports.loggedIn = (req, res, next) => {
+const loggedIn = (req, res, next) => {
```

Comment:
Assigning these arrow functions to actual variables gives them names in the stack trace, which turns out to be terribly useful.

Reply:
yes, i'm a big fan of this small change